flink-clients/src/main/java/org/apache/flink/client/deployment/executors/PipelineExecutorUtils.java

```diff
@@ -42,10 +42,13 @@ public class PipelineExecutorUtils {
      * @param configuration the configuration with the necessary information such as jars and
      *     classpaths to be included, the parallelism of the job and potential savepoint settings
      *     used to bootstrap its state.
+     * @param userClassloader the classloader used to load class during compile JobGraph
```

Comment:
maybe "the classloader which can load user classes", I think it's more accurate because currently it's the system classloader which is mainly used to build JobGraph.